src/app/common/modals/userinput.less

```diff
@@ -11,3 +11,9 @@
         }
     }
 }
+
+.markdown {
+    font-family: var(--base-font);
```

Comment:
updated how --base-font works.  now just write `font: var(--base-font)` and you get the font-family, size, weight, etc.

Reply:
good point--just pushed a fix